CMakeLists.txt

```diff
@@ -11,4 +11,4 @@ set(CXX_SUGGEST_OVERRIDE ON
     FORCE)
 
 # Declare project name and version
-elements_project(Alexandria 2.25 USE Elements 6.0.1)
+elements_project(Alexandria 2.25 USE Elements 6.0.1 DESCRIPTION "SDC-CH common library for the Euclid project ")
```

Comment:
there is a trailing space at the end of the description string, is that on purpose?

Reply:
A typo. Fixed.